src/patternfly/demos/Tabs/examples/Tabs.md

```diff
@@ -0,0 +1,813 @@
+              <p>Additional architectural content</p>
+            {{/tab-content}}
+          {{/example-wrapper}}
+        {{/grid-item}}
+      {{/grid}}
+    {{/tab-content}}
+    {{#> tab-content tab-content--attribute=(concat 'aria-labelledby="' tab-content--id '-knowledge-link" id="' tab-content--id '-knowledge-panel"')}}
+      {{#> content}}
+        <p>Knowledge content</p>
+      {{/content}}
+    {{/tab-content}}
+    {{#> tab-content tab-content--attribute=(concat 'aria-labelledby="' tab-content--id '-support-link" id="' tab-content--id '-support-panel"')}}
+      {{#> content}}
+        <p>Support content</p>
+      {{/content}}
+    {{/tab-content}}
+  {{/page-main-section}}
+{{/tabs--page-wrapper}}
+```
+
+### Vertical tabs
+```hbs isFullscreen
+{{#> tabs--page-wrapper tabs--page-wrapper--id="vertical-tabs-example"}}
+  {{> page-template-breadcrumb page-template-breadcrumbs--IsOS="true"}}
+  {{#> page-main-section page-main-section--IsLimitWidth="true" page-main-section--modifier="pf-m-light"}}
+    {{#> title title--modifier="pf-m-2xl pf-u-mt-md"}}
+      Install OpenShift on AWS
+    {{/title}}
+  {{/page-main-section}}
+  {{#> page-main-section page-main-section--IsLimitWidth="true" tabs--id=(concat tabs--page-wrapper--id '-tabs') tab-content--id=(concat tabs--page-wrapper--id '-tabs')}}
+    {{#> card card--modifier="pf-m-fill"}}
+      {{#> sidebar}}
+        {{#> sidebar-panel}}
+          {{#> tabs tabs--modifier="pf-m-inset-md pf-m-inset-lg-on-xl pf-m-vertical pf-m-box" }}
+            {{#> tabs-list}}
+              {{> __tabs-item
+                __tabs-item--current="true"
+                __tabs-item--id="installation"
+                __tabs-item--aria-label="Installation"
+                __tabs-item--text="Installation"
+                __tabs-item--attribute=(concat 'aria-controls="' tabs--id '-installation-panel"')}}
+              {{> __tabs-item
+                __tabs-item--id="limits"
+                __tabs-item--aria-label="Limits"
+                __tabs-item--text="Limits"
+                __tabs-item--attribute=(concat 'aria-controls="' tabs--id '-limits-panel"')}}
+              {{> __tabs-item
+                __tabs-item--id="cluster-installation"
+                __tabs-item--aria-label="Cluster installation"
+                __tabs-item--text="Cluster installation"
+                __tabs-item--attribute=(concat 'aria-controls="' tabs--id '-cluster-installation-panel"')}}
+              {{> __tabs-item
+                __tabs-item--id="subscription-support"
+                __tabs-item--aria-label="Subscription and support"
+                __tabs-item--text="Subscription and support"
+                __tabs-item--attribute=(concat 'aria-controls="' tabs--id '-subscription-support-panel"')}}
+            {{/tabs-list}}
+          {{/tabs}}
+        {{/sidebar-panel}}
+        {{#> sidebar-content sidebar-content--modifier="pf-u-p-lg"}}
```

Comment:
It would be great if we can avoid using a utility class in the demo. Maybe the sidebar should have padding like the drawer?

Reply:
Technically you could use a page section. The only thing about page sections is that they expect to be in a flex layout as flex children. Putting them in `.pf-c-page__main-group` is a way to introduce them somewhere other than a direct child of `.pf-c-page__main`. Though I think the sidebar should probably have padding and that seems like a suitable approach.